components/sst_importer/src/sst_importer.rs

```diff
@@ -230,6 +230,7 @@ impl SstImporter {
         dst_file: std::path::PathBuf,
         backend: &StorageBackend,
         expect_sha256: Option<Vec<u8>>,
+        use_kms: bool,
```

Comment:
please Leave a TODO here, and add some comments about why we cannot encrypt kv file 🤔